src/style.css

```diff
@@ -1 +1,57 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  width: 140px;
+  height: 160px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
+  height: 116px;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin: 2px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-image: url(../../layout_stars/src/images/star.svg);
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(1) {
+  background-image: url(../../layout_stars/src/images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(-N + 1) {
+  background-image: url(../../layout_stars/src/images/star-active.svg);
+}
```

Comment:
The selector `.stars--1 .stars__star:nth-child(-N + 1)` is incorrect. It should be `.stars--1 .stars__star:nth-child(-n + 1)` to ensure the correct star is highlighted. This applies to other similar selectors as well.